packages/core/saved-objects/core-saved-objects-api-server-internal/src/mocks/saved_objects_extensions.mock.ts

```diff
@@ -47,7 +46,7 @@ const createSpacesExtension = (): jest.Mocked<ISavedObjectsSpacesExtension> => (
   getSearchableNamespaces: jest.fn(),
 });
 
-const create = (): jest.Mocked<SavedObjectsExtensions> => ({
```

Comment:
NIT: `jest.Mocked<SavedObjectsExtensions>` was shallowing the underlying mock definitions, so I removed the explicit signature to use return type inference instead.

Reply:
TIL